src/models/searchservice/response/AddressDirectAnswer.ts

```diff
@@ -0,0 +1,28 @@
+import { FieldValueDirectAnswer } from './FieldValueDirectAnswer';
+import { BuiltInFieldType } from './BuiltInFieldType';
+
+/**
+ * An interface for direct answers for a {@link FieldValueDirectAnswer} with address values.
+ *
+ * @public
+ */
+export interface AddressDirectAnswer extends FieldValueDirectAnswer<Address> {
+  fieldType: BuiltInFieldType.Address
+}
+
+/**
+ * A type interface with address fields to use in {@link FieldValueDirectAnswer.value}.
+ *
+ * @public
+ */
+export interface Address {
```

Comment:
Out of curiosity are all of these properties truly optional?

Reply:
I tested it out on slanswers-test experience and I think line 1, city, region, and postalCode are required, although they are generated through CLI as optionals.

Reply:
What makes you think they're required?

Reply:
When you add address to a person or a location, these fields are required.